cmd/influx/cli/cli.go

```diff
@@ -642,6 +644,114 @@ func (c *CommandLine) writeColumns(response *client.Response, w io.Writer) {
 	}
 }
 
+func (c *CommandLine) writeLineProtocol(response *client.Response, w io.Writer) {
+	for _, result := range response.Results {
+		// Print out all messages first
+		for _, m := range result.Messages {
+			fmt.Fprintf(w, "%s: %s.\n", m.Level, m.Text)
+		}
+
+		for _, multiRow := range result.Series {
+			for _, row := range c.formatLineProtocol(multiRow) {
+				fmt.Fprintln(w, row)
+			}
+		}
+	}
+}
+
+func (c *CommandLine) formatLineProtocol(row models.Row) []string {
```

Comment:
Did you look into using the existing serializer ([`"github.com/influxdata/influxdb".Point.String()`](github.com/influxdata/influxdb))? Would make the code much smaller, and be better to not have multiple serializers for the same format.